information.txt

```diff
@@ -48,7 +52,7 @@ Overall evaluation: 2 (accept)
 ----------- Overall evaluation -----------
 The paper is about the experience of the authors in organizing a workshop focused on the analysis and interpretation of neu- roimaging data, ranging from whole-brain functional magnetic res- onance imaging (fMRI) to single-cell microscopy.
 
-They used a cloud computing services and advanced cyberinfrastructure support for improving learning ex-periences in bootcamp-style events. I found the paper interesting, although it should be more general (some of the proposed solutions are specific for their campus).
+They used a cloud computing services and advanced cyberinfrastructure support for improving learning ex-periences in bootcamp-style events. I found the paper interesting, although it should be more general (some of the proposed solutions are specific for their campus).  <--- Hopefully addressed by the paragraph on the XSEDE cluster being widely available etc.
```

Comment:
I couldnt find this revision or it was not highlighted enough. But I think in general, adding a sentence that says something like this would be good:

"The container technologies (e.g. Docker) and cloud computing resources (e.g. XSEDE) are widely available platforms. However, this teaching environment can be easily applied using a different container or cloud computing resource."